bigtable/bttest/inmem.go

```diff
@@ -61,7 +62,7 @@ const (
 	minValidMilliSeconds = 0
 
 	// MilliSeconds field of the max valid Timestamp.
-	maxValidMilliSeconds = int64(time.Millisecond) * 253402300800
+	maxValidMilliSeconds = math.MaxInt64 - math.MaxInt64%1000
```

Comment:
can you please add why this value (`math.MaxInt64 - math.MaxInt64%1000`) is used in a comment?

Reply:
Done